### DISCUSSION TITLE
How to add images to Asset Manager?

### DISCUSSION THREAD

**Discussion by denys-fominykh** [GrapesJS/grapesjs#5600] [Q&A]
I use React + TS + GrapesJS 0.21.7 + grapesjs-preset-newsletter 1.0.2. When I try to add my images using `editor.AssetManager.add()`, I see that my images are in the config (if `editorObj.AssetManager.getConfig()` is called), but there is nothing in the “Select Images” window. What could be the problem? In the GrapesJS config file, the `assetsManager` field is set to `custom: true`.

```
const assetManager = editor.AssetManager;

    assetManager.add([
      {
        src: 'https://www.topgear.com/sites/default/files/cars-car/image/2022/11/308392_2020_civic_type_r_mmc_launch_2.jpeg',
        name: 'Image Name 1',
        type: 'image',
        height: 1688,
        width: 3000,
      },
      {
        src: 'https://cdn.arstechnica.net/wp-content/uploads/2023/05/2023-Honda-CR-V-Hybrid-1-800x600.jpg',
        name: 'Image Name 2',
        type: 'image',
        height: 600,
        width: 800,
      },
    ]);
```
<img width="386" alt="Screenshot 2024-01-08 at 12 35 07" src="https://github.com/GrapesJS/grapesjs/assets/28676747/82cc12a1-14cd-45e9-939b-c131a89e6ba2">

**Comment by denys-fominykh** [comment]
<img width="930" alt="Screenshot 2024-01-08 at 12 35 17" src="https://github.com/GrapesJS/grapesjs/assets/28676747/d235b78f-769a-4b86-8ce6-9624c9b9ee09">

**Comment by artf** [comment]
>  In the GrapesJS config file, the assetsManager field is set to custom: true.

Well, that's the reason... that option means "don't render the built-in AssetManager" https://grapesjs.com/docs/modules/Assets.html#customization
